src/pages/users/users-page/components/user-tab/user-tab.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { push } from 'connected-react-router';
+import TableContainerGenerator from '../../../../../containers/table-container-generator';
+import TableContainerRow from '../../../../../containers/table-container-row';
+import { userRoleTranslations } from '../../../../../translations/user.translations';
+import { formatPhoneNumber } from '../../../../../utils/format-phone-number';
+import { config } from '../../../../../configs';
+
+const tableHeaders = config.tableHeadRowTitles.users.userTab;
+
+const UserTab = (props) => {
+  const { list, onDelete } = props;
+  const dispatch = useDispatch();
+
+  const usersItems = list.map((userItem, index) => (
+    <TableContainerRow
+      key={index}
```

Comment:
Don't use index for key, use something stable like **id** or **name**

Reply:
fixed